crates/wasm-backend-traits/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
```

Comment:
I suggest to remove all Wasmer related stuff from this PR and add it later in a separate PR.